### ISSUE TITLE
Missing alternative text on banner image

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#806]
Even though `aria-label="Lady Bird Lake walking trail"` is set on the `class="coa-HeroHome"` div, the image itself does not have alt text.

- https://alpha.austin.gov/
- https://alpha.austin.gov/police-oversight

OPO site:
Banner image - Aerial photo of the Austin skyline looking over from Town Lake

**URL**
Any page with a banner/hero image, including alpha.austin.gov.

**Screenshots**
<img width="759" alt="screen shot 2018-10-11 at 10 07 20 am" src="https://user-images.githubusercontent.com/1463708/46815417-83584000-cd40-11e8-8976-85e34015c5c9.png">

**Additional context**
Results from AChecker:[achecker_2018-10-11_15-27-49.pdf](https://github.com/cityofaustin/techstack/files/2469589/achecker_2018-10-11_15-27-49.pdf)

Aha! Link: https://desi1.aha.io/features/RT-178

**Comment by briaguya** [comment]
This should be a one line fix in our ResponsiveImage component, just need to add the alt text to the <img> we're returning there

**Comment by briaguya** [comment]
This is actually quite complex. In order to have both responsive images and the overlay with text, we have a <div> with the role of "img" in there. The recommended way to handle this is to have an aria-label set on it instead of using alt text as we aren't actually using an <img> tag, so we're following the pattern here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img

This won't get WAVE to pass, and even manually setting an alt attribute on the div (not something that's commonly done anywhere) fails in WAVE.

**Comment by amenity** [comment]
Look in `coa-HeroHome__container` - might not actually be this bad.

**Comment by amenity** [comment]
<img width="1169" alt="screen shot 2019-02-27 at 9 33 05 am" src="https://user-images.githubusercontent.com/1463708/53502057-eb4ebf00-3a72-11e9-8d4e-f924f6d03f7d.png">
<img width="1170" alt="screen shot 2019-02-27 at 9 33 17 am" src="https://user-images.githubusercontent.com/1463708/53502059-ebe75580-3a72-11e9-87a4-875ad63256ba.png">
